app/src/test/java/nerd/tuxmobil/fahrplan/congress/schedule/ConferenceTest.kt

```diff
@@ -30,6 +30,32 @@ class ConferenceTest {
         assertThat(timeZoneOffset).isEqualTo(ZoneOffset.of("+02:00"))
     }
 
+    @Test
```

Comment:
GitHub does not showing the [former unresolved discussion](https://github.com/EventFahrplan/EventFahrplan/commit/d2695c66ef03bc61b963f45e05b38b211f1406bd#r47568078). Let take this comment as a pointer.

Reply:
@a-tze Not sure if you received this message because I put your handle **after** I wrote the initial commit. Otherwise, sorry for the spam!

Reply:
Well.. not sure what the state is now, I can hardly follow this PR anymore ;) but it looks good, my only real recent contribution was the idea with using October's DST change.

Reply:
And that was/is very welcome. You can just look at the test scenarios if you like to see if they make sense.

Reply:
@a-tze If I understood you right [here](https://github.com/EventFahrplan/EventFahrplan/pull/367#discussion_r584186773) we can resolve this comment now. Closing here.